weblate/templates/snippets/query-builder.html

```diff
@@ -34,11 +34,11 @@
               <div class="btn-group">
                 <button type="button" class="btn btn-default dropdown-toggle search-field" data-field="has:suggestion" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span class="search-label">{% trans "String has suggestion" %}</span> <span class="caret"></span></button>
                 <ul class="dropdown-menu">
-                  <li><a href="#" data-field="has:suggestion">{% trans "String has suggestion" %}</a></li>
-                  <li><a href="#" data-field="has:comment">{% trans "String has comment" %}</a></li>
-                  <li><a href="#" data-field="has:failing_check">{% trans "String has failing check" %}</a></li>
-                  <li><a href="#" data-field="has:plural">{% trans "String has plural" %}</a></li>
-                  <li><a href="#" data-field="has:context">{% trans "String has context" %}</a></li>
+                  <li><a href="#" data-field="has:suggestion">{% trans "String suggestion exists" %}</a></li>
+                  <li><a href="#" data-field="has:comment">{% trans "Commented string" %}</a></li>
+                  <li><a href="#" data-field="has:failing_check">{% trans "Failing checks for string" %}</a></li>
+                  <li><a href="#" data-field="has:plural">{% trans "Pluralized string" %}</a></li>
+                  <li><a href="#" data-field="has:context">{% trans "Contextualized string" %}</a></li>
```

Comment:
This reminds me that this should be generated from `weblate/trans/filter.py` so it's more important to change the strings there.

Reply:
The ones in filter.py were just plain better, the ones in here can be overwritten. "Pluralized string" isn't in there though.

Reply:
I've switched it in c3283e2e51e7ad70473ab8d747fee0d7641ff200